tests/test_buildozer.py

```diff
@@ -155,15 +156,17 @@ def test_android_ant_path(self):
 
         # Mock first run
         with mock.patch('buildozer.Buildozer.download') as download, \
-                mock.patch('buildozer.Buildozer.file_extract') as extract_file, \
+                mock.patch('buildozer.Buildozer.file_extract') as m_file_extract, \
                 mock.patch('os.makedirs'):
             ant_path = target._install_apache_ant()
-            assert ant_path == my_ant_path
```

Comment:
This is still there, I just to it outside the context manager because it didn't need to be